article-analyzer/src/main/java/co/inventorsoft/academy/spring/articleanalyzer/repository/ArticleRepository.java

```diff
@@ -0,0 +1,45 @@
+package co.inventorsoft.academy.spring.articleanalyzer.repository;
+
+import co.inventorsoft.academy.spring.articleanalyzer.model.Article;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+
+@Repository("articleRepository")
+public class ArticleRepository {
+    private final Gson gson;
+    private final String filePath;
```

Comment:
just use @value directly here over the filed, it will be better then via constructor param